articles/multifactor-authentication/guardian/user-guide.md

```diff
@@ -0,0 +1,79 @@
+
+![](/media/articles/mfa/guardian-recovery-code.png)
+
+### Google Authenticator
+If you would prefer to use the Google Authenticator app, click on the link for Google Authenticator.  You will need to download Google Authenticator for [Android](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2) or [iOS](https://itunes.apple.com/us/app/google-authenticator/id388497605)
+
+[Click here to learn more about Google Authenticator](https://support.google.com/accounts/answer/1066447) 
+
+A code will appear, and you will have five minutes to scan the code before it expires. After scanning the code, you will get a six digit code to enter. Once you enter this code, you will see a confirmation screen which has a recovery code. If for some reason you do not have your mobile device, you will need this recovery code to login. Make sure to take note of this code and keep it somewhere safe. Check the box that you have recorded the code, and then you are logged in.
+
+![](/media/articles/mfa/google-code.png)
+
+### SMS
+
+Depending on the applications settings, SMS may be an option to use for two-factor authentication. Click on the the **SMS** link after entering your email and password.
+
+Then select your phone number's country code and enter your mobile phone number. You must be able to receive SMS to your device to use this option.
+
+![](/media/articles/mfa/sms.png)
+
+Then you should receive a six digit code in a message to your phone. Enter this code into the box. Then you will see a recovery code, make sure to note this code as you will need it to login if you do not have your device. Check the box that you have recorded the code, and then you are all set and logged in.
+
+## Logging in
+
+Depending on the type of two-factor authentication you chose when you were signing up, this will affect how you login.
+
+### Guardian
+
+After entering your username and password, a push notification will be sent to the Guardian app on your mobile device. This notification is a login request, it includes the application name, the OS and browser of the request, the location and the date of the request. If you recognize this request as your own, tap the **Allow** button. You should now be logged in.
+
+### Google Authenticator
+
+After entering your username and password, you will be prompted for a six digit code. Open the Google Authenticator app on your mobile device to find the correct code. If you use Google Authenticator for other applications as well, make sure you are using the code for the current application. After entering the six digit code you will be logged in.
+
+### SMS
+
+After entering you enter your email and password, a SMS message will be sent to the phone number you entered when you signed up. Enter the six digit code from the message into the box to complete your login.
+
+## Troubleshooting
```

Comment:
I think we should also talk about rate limits and transaction expiration here maybe with some screenshot on how to diagnose the problem